index.html

```diff
@@ -350,36 +197,980 @@ <h6 class="mdc-list-group__subheader">Friends of CD</h6>
+                          Challenges
+                        </span>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-6-desktop mdc-layout-grid__cell--span-4-phone mdc-layout-grid__cell--span-4-tablet">
+          <div class="mdc-card mdc-elevation-transition projects-tab-card">
+            <div class="mdc-card__media mdc-card__media--16-9 app_website_site_card_image"></div>
+            <div class="padded-projects-card">
+              <h2 class="projects-card-title mdc-typography mdc-typography--headline6">App Website</h2>
+            </div>
+            <div class="projects-card-secondary mdc-typography mdc-typography--body2">Companion Website for the unofficial Cyber Discovery App. Essentially all the features of the App (Link in menu), but in a web-based form. No downloading required.</div>
+            <div class="mdc-card__actions">
+              <div class="mdc-card__action-buttons">
+                <button class="mdc-button mdc-card__action mdc-card__action--button" onclick="loadNewPage('https://github.com/CyberDiscovery/Cyber-Discovery-App-Website');">Repo</button>
+                <button class="mdc-button mdc-card__action mdc-card__action--button" onclick="loadProjectDetails('Cyber-Discovery-App-Website');">More</button>
+              </div>
+              <div class="mdc-card__action-icons">
+                <div class="mdc-menu-surface--anchor">
+                  <button class="mdc-icon-button material-icons mdc-card__action mdc-card__action--icon--unbounded project-menu-btn" title="More options" data-mdc-ripple-is-unbounded="true" id="btn-proj-6">more_vert</button>
+                  <div class="mdc-menu mdc-menu-surface" id="menu-proj-6">
+                    <ul class="mdc-list" role="menu" aria-hidden="true" aria-orientation="vertical">
+                      <li class="mdc-list-item" role="menuitem" onclick="loadNewPage('https://github.com/login?return_to=%2FCyberDiscovery%2FCyber-Discovery-App-Website');">
+                        <span class="mdc-list-item__text">
+                          <i class="material-icons project_card_menu_icons">call_split</i>
+                          Fork Project
+                        </span>
+                      </li>
+                      <li class="mdc-list-item" role="menuitem" onclick="loadNewPage('https://github.com/CyberDiscovery/Cyber-Discovery-App-Website/issues');">
+                        <span class="mdc-list-item__text">
+                          <i class="material-icons project_card_menu_icons">view_list</i>
+                          View Issues
+                        </span>
+                      </li>
+                      <li class="mdc-list-item" role="menuitem" onclick="loadNewPage('https://cyber-discovery-app.firebaseapp.com/');">
```

Comment:
```suggestion
                      <li class="mdc-list-item" role="menuitem" onclick="loadNewPage('https://app.cyberdiscoverycommunity.uk/');">
```

Reply:
Also, this appears to be semantically a hyperlink. Thus, it should probably have the appropriate ARIA role (and I think it isn't a menuitem, especially because this appears to be a navigation segment of the page - which from a brief search should be handled with its specific ARIA role).

The way I normally see navigation hyperlinks is with a DOM structure of NAV > LI > A. Of course, it doesn't matter how we implement the navigation section so long as it is accessible, and proper use of ARIA will help with that.

I haven't had a look at the rewrite but it looks really thorough and big and amazing, thank you so much!

Reply:
@joker314 You know, you're right. I am confused by the documentation. Perhaps its best to remove the `role=` attribute here? I mean an earlier version of the framework used this and I haven't updated it since and they've removed all references to this attribute. :confused: 

https://material.io/develop/web/components/lists/

https://github.com/material-components/material-components-web/tree/master/packages/mdc-list